fe/fe-core/src/main/java/org/apache/doris/common/proc/TablesProcDir.java

```diff
@@ -128,8 +129,8 @@ public ProcResult fetchResult() throws AnalysisException {
                     tableInfo.add(FeConstants.null_string);
                     tableInfo.add(replicaCount);
                 }
-
-
+                long lastUpdateTime = table.getLastUpdateTime();
+                tableInfo.add(lastUpdateTime > 0 ? TimeUtils.longToTimeString(lastUpdateTime) : "UNRECORDED");
```

Comment:
maybe lastUpdateTime is -1L，Can the TimeUtils.longToTimeString(-1)  instead of "UNRECORDED"?